src/badge_hub.py

```diff
@@ -68,7 +68,7 @@
 def get_devices(device_file):
```

Comment:
Please remove this function fro mbadge_hup.py. We aren't using it anymore (the devices are loaded by the standalone manager)